packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -176,6 +212,247 @@ - (void)testSeekToInvokesTextureFrameAvailableOnTextureRegistry {
   XCTAssertEqual([player position], 1234);
 }
 
```

Comment:
We should have individual unit test files for each specific cache class, in addition to these higher-level native tests.